app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -41,16 +58,19 @@ public void onResume() {
         setParallelDownloadsText(UserPreferences.getParallelDownloads());
     }
 
-    private void setupNetworkScreen() {
+    private void setupNetworkScreen()
+    {
```

Comment:
AntennaPod uses Google's Java code style. Braces open in the same line (like it was before).